pkg/cloud/libvirt/client/s390_ignition.go

```diff
@@ -1,25 +1,27 @@
 package client
 
 import (
-	"bytes"
 	"fmt"
-	"io"
+	"os"
 	"os/exec"
-	"strings"
+	"io"
+	"io/ioutil"
+	"path/filepath"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 
+	libvirt "github.com/libvirt/libvirt-go"
 	"github.com/golang/glog"
 	libvirtxml "github.com/libvirt/libvirt-go-xml"
 	providerconfigv1 "github.com/openshift/cluster-api-provider-libvirt/pkg/apis/libvirtproviderconfig/v1beta1"
-	"github.com/pkg/errors"
+
 )
 
 var execCommand = exec.Command
 
 func setIgnitionForS390X(domainDef *libvirtxml.Domain, client *libvirtClient, ignition *providerconfigv1.Ignition, kubeClient kubernetes.Interface, machineNamespace, volumeName string) error {
-	glog.Info("Creating ignition file for s390x")
+	glog.Infof("Creating ignition file for s390x")
```

Comment:
yes, unrelated change..., I will cancel this change.

Reply:
please don't. The right thing to do would be to separate it in another patch.

Reply:
Um, glog.Infof and glog.Info are both OK, right? I changed it because I want to unify to use glog.Infof in the file s390_ignition.go. 
Do I need to create a new PR to change this sentence? Or use glog.Infof directly here?

Reply:
>  Do I need to create a new PR to change this sentence? Or use glog.Infof directly here?

No, I didn't say separate PR. :) I said, separate patch and yes, ideally every logical change (no matter how small) should go in a separate patch.

Reply:
ok~, and could I submit a new commit in this PR? right?

Reply:
I separated the patch by a new commit. any other advise pls let me know, thank you ~